src/Wallabag/CoreBundle/Controller/EntryController.php

```diff
@@ -52,6 +58,22 @@ public function addEntryFormAction(Request $request)
     /**
      * @param Request $request
      *
+     * @Route("/w", name="new_bookmarklet")
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function addEntryViaBookmarklet(Request $request)
```

Comment:
Because I had to define a route.

Envoyé depuis mon appareil Android.

-----Original Message-----
From: Jeremy Benoist <notifications@github.com>
To: wallabag/wallabag <wallabag@noreply.github.com>
Cc: "Nicolas Lœuillet" <nicolas@loeuillet.org>
Sent: mer., 07 oct. 2015 22:50
Subject: Re: [wallabag] implement bookmarklet (#1471)

> @@ -52,6 +58,22 @@ public function addEntryFormAction(Request $request)
>      /**
>       * @param Request $request
>       *
> +     * @Route("/w", name="new_bookmarklet")
> +     *
> +     * @return \Symfony\Component\HttpFoundation\Response
> +     */
> +    public function addEntryViaBookmarklet(Request $request)

You might need a test for that action.
And shouldn't this method be `addEntryViaBookmarkletAction` instead?

And why `/w` ?

---
Reply to this email directly or view it on GitHub:
https://github.com/wallabag/wallabag/pull/1471/files#r41441623